src/oph/ehoks/palaute.clj

```diff
@@ -191,6 +194,12 @@
       (hoks/tuva-related? hoks)
       [:ei-laheteta :tuva-opiskeluoikeus-oid :tuva-opiskeluoikeus]
 
+      ;; this is for the transition period with herätepalvelu
+      (and existing-ddb-herate (seq @existing-ddb-herate))
+      [:heratepalvelussa herate-date-field :heratepalvelun-vastuulla]
```

Comment:
[Oli puhetta](https://opetushallitus.slack.com/archives/C03JN1DGD53/p1742211619839369?thread_ts=1742202509.202229&cid=C03JN1DGD53) tosiaan, että jättäis luomatta palautteet jotka löytyy jo Herätepalvelusta. Eipä tästä `heratepalvelussa`-tilasta varmaan suurta haittaakaan ole, dokumentointimielessä ehkä ihan hyväkin? :thinking:

Reply:
Joo tosiaan, tää on nyt eri tavalla kuin puhuin.  En ehtinyt vielä selittää mut syy miksi tein tän näin on se, että muuten vanhoja palautteita luova eräajo (`palaute/reinit-palautteet-for-uninitiated-hokses!`) ei pysty oikein merkitsemään mitenkään että jokin palaute on jo tarkistettu, jolloin se päätyy tarkistamaan uudestaan ja uudestaan samoja HOKSeja.

Eli tavallaan täs nyt sitten se ettei palautetta oo tarkoittaa "me ei tiedetä mikä tän palautteen tilanne on" ja `heratepalvelussa` tarkoittaa "me tiedetään että tää palaute on herätepalvelussa ja sen vastuulla".

Sorry etten selittäny aiemmin, unohdin.